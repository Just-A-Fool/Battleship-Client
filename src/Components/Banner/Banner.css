.banner {
  background-image: url('../../Img/deep-ocean.jpg');
  color: white;
  display: flex;
  justify-content: center;
  top: 0;
  left: 0;
  right: 0;
  overflow: hidden;
  position: fixed;
  width: 100%;
}