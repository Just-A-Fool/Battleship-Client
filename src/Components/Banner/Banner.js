import React, { Component } from 'react';
import './Banner.css';

class Banner extends Component {

  render() {
    return (
      <div className='banner'>
        <h1>BATTLESHIP</h1>
      </div>
    )
  }
};

export default Banner;


