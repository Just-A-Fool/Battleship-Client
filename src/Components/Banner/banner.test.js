/* eslint-disable no-unused-expressions */
import React from 'react'
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Banner from './Banner'

describe(`Banner Component`, () => {
    it(`Renders without errors`, () => {
        const div = document.createElement('div');
        ReactDOM.render(<Banner /> , div)
        ReactDOM.unmountComponentAtNode(div)
    })

    it(`Displays the headline 'BATTLESHIP'`, () => {
        const wrapper = shallow(<Banner />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })

    it(`Contains an 'h1' element with headline`, () => {
        const wrapper = shallow(<Banner />)
        expect(wrapper.contains(<h1>BATTLESHIP</h1>)).toEqual(true)
    });
});

