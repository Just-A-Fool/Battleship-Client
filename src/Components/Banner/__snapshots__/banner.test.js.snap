// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner Component Displays the headline 'BATTLESHIP' 1`] = `
<div
  className="banner"
>
  <h1>
    BATTLESHIP
  </h1>
</div>
`;
